Constraints/ViewController.swift

```diff
@@ -7,13 +7,43 @@
 
 import UIKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, UITableViewDataSource, UITextViewDelegate {
+    
+    // MARK: Outlets
+    @IBOutlet weak var tableView: UITableView!
+    
+    var tableViewData: [MessageData] = []
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        tableViewData = DataLoader().loadMessages()
+        
+        tableView.register(UITableViewCell.self,
```

Comment:
this line is not needed

Reply:
https://github.com/agustingodoyc/Constraints/pull/3/commits/cf7bd49e01a4de1765fca2b2d9859ea67728b93f

Reply:
https://github.com/agustingodoyc/Constraints/pull/3/commits/4d492815c7dee4cf4d6ed2a22942dde057df03da